app/models/host/base.rb

```diff
@@ -16,7 +16,7 @@ class Base < ApplicationRecord
     friendly_id :name
 
     validates_lengths_from_database
-    belongs_to :model, :name_accessor => 'hardware_model_name'
+    belongs_to :hardware_model, class_name: 'Model', foreign_key: 'model_id'
```

Comment:
This is not the same - the `model` and `model=` methods are still used. It's also part of the REST API.